pkg/notify/notify_test.go

```diff
@@ -174,6 +174,19 @@ func testReceiverConfig2() *config.ReceiverConfig {
 	}
 }
 
+func testReceiverConfig3() *config.ReceiverConfig {
```

Comment:
nit: It looks like you're following the convention, but I guess it would be much better to give a more meaningful name to this, e.g
```suggestion
func testReceiverConfigAutoResolve() *config.ReceiverConfig {
```